apps/website/src/content/docs/informationpanel.mdx

```diff
@@ -17,6 +14,33 @@ The information panel is a pattern used primarily in tables. It allows viewing o
 
 The info panel is reserved for complex tables, because it allows the user to gradually explore the content without overwhelming them with too many details. It also helps avoiding the unpleasant horizontal scrolling when the table gets too wide. Not every table requires an information panel.
```

Comment:
Some of the intro text needs to be rewritten/trimmed to be less opinionated and more accurately reflect what the component does.